certbot/colored_logging.py

```diff
@@ -1,45 +0,0 @@
-"""A formatter and StreamHandler for colorizing logging output."""
-import logging
-import sys
-
-from certbot import util
-
-
-class StreamHandler(logging.StreamHandler):
```

Comment:
Would you mind terribly if this stayed in its own file? We do like random bits of OOO sprinkled throughout, and it's nice for them to stay in their own files like a proper OOO paradigm.